package.json

```diff
@@ -16,12 +18,11 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
-    "@babel/preset-env": "^7.1.0",
-    "jasmine-focused": "^1.0.4",
-    "joanna": "0.0.11",
-    "rimraf": "^2.2.2",
-    "temp": "^0.6.0"
+    "@babel/cli": "^7.12.1",
+    "@babel/core": "^7.12.3",
+    "babel-preset-atomic": "^2.5.4",
+    "cross-env": "^7.0.2",
+    "jasmine-focused": "^1.0.7",
+    "joanna": "https://github.com/aminya/joanna"
```

Comment:
It is not clear why we are updating joanna. Could you shade some light on this please.

Reply:
Joana parses JavaScript and generates documentation. I updated it in my fork, so I can use the new JavaScript syntax here. Without this, it will fail to parse. Although, not sure if the docs are used anywhere.

https://github.com/atom/joanna/pull/12

Reply:
I don't think the docs are used anywhere as well. Your [PR](https://github.com/atom/joanna/pull/12) has been open for now close to 3 months and no communication from the maintainer. 

I don't mind adding `https://github.com/aminya/joanna` as a dependancy.